appinventor/blocklyeditor/src/msg/en/_messages.js

```diff
@@ -110,7 +109,7 @@ Blockly.Msg.en.switch_language_to_english = {
     Blockly.ERROR_BREAK_ONLY_IN_LOOP = "The break block should be used only within loops";
 
 // Colour Blocks.
-    Blockly.Msg.LANG_COLOUR_PICKER_HELPURL = 'http://appinventor.mit.edu/explore/ai2/support/blocks/colors#basic';
+    Blockly.Msg.LANG_COLOUR_PICKER_HELPURL = '/reference/blocks/colors#basic';
```

Comment:
Missing the .html on this URL, which results in a 404.

Reply:
This was true of all the block links previously. I'm not entirely sure how that worked, but it clearly did work when pointing to the other repository.

Reply:
appinventor.mit.edu is hosted by GitHub pages. It might be that they are doing HTTP content negotiation with the browser and determining that serving the HTML file is safe given that there are no other alternatives, whereas App Engine isn't.